pycbc/workflow/coincidence.py

```diff
@@ -753,4 +753,4 @@ def setup_multiifo_combine_statmap(workflow, final_bg_file_list, out_dir, tags):
                                                  cluster_window,
                                                  tags)
     workflow.add_node(combine_statmap_node)
-    return combine_statmap_node.output_files
+    return combine_statmap_node.output_file
```

Comment:
@spxiwh Everywhere else the .output_files is used, does this supposed to return 1 file, the others a filelist? If yes I will approve this though I would not mind if this was tested to be sure this is how it works.

Reply:
`job.output_file` will return only one file if the job has only one output file (it will fail if there are more than one)

So we should use that if we know there is only one output.

This has been tested in a workflow (see SLACK for the link to a result page with this enabled).